android/app/src/test/java/org/smartregister/fhircore/util/UtilsTest.kt

```diff
@@ -22,7 +22,10 @@ import org.apache.commons.lang3.time.DateUtils
 import org.joda.time.DateTime
 import org.junit.Assert
 import org.junit.Test
+import org.robolectric.annotation.Config
+import org.smartregister.fhircore.shadow.FhirApplicationShadow
 
+@Config(shadows = [FhirApplicationShadow::class])
```

Comment:
We probably don't need this since this test does not use Robolectric/Junit4 Runner